src/strategies/base.ts

```diff
@@ -428,6 +428,30 @@ export abstract class BaseStrategy implements Strategy {
               );
           }
         }
+      } else if (extraFile.endsWith('.json')) {
+        extraFileUpdates.push({
+          path: this.addPath(extraFile),
+          createIfMissing: false,
+          updater: new GenericJson('$.version', version),
+        });
+      } else if (extraFile.endsWith('.yaml') || extraFile.endsWith('.yml')) {
+        extraFileUpdates.push({
+          path: this.addPath(extraFile),
+          createIfMissing: false,
+          updater: new GenericYaml('$.version', version),
+        });
+      } else if (extraFile.endsWith('.toml')) {
+        extraFileUpdates.push({
+          path: this.addPath(extraFile),
+          createIfMissing: false,
+          updater: new GenericToml('$.version', version),
+        });
+      } else if (extraFile.endsWith('.xml')) {
+        extraFileUpdates.push({
+          path: this.addPath(extraFile),
+          createIfMissing: false,
+          updater: new GenericXml('/*/version', version),
+        });
```

Comment:
This is potentially a breaking change for users. These file types support the `Generic` updater's annotations. We do have a `CompositeUpdater` which we should probably use both the `Generic` and `Generic<Format>` updaters. This allows substituting both the exact version field and any additional annotated values.